src/components/Settings/AllSettingsFields.tsx

```diff
@@ -271,7 +271,7 @@ export const AllSettingsFields = forwardRef(
               , and start a discussion if you don't see it! Your feedback will
               help us prioritize what to build next.
             </p>
-            {PACKAGE_NAME.indexOf('-nightly') === -1 && (
+            {IS_NIGHTLY && (
```

Comment:
As pointed out by @nadr0 this needs to be

```suggestion
            {!IS_NIGHTLY && (
```